app/models/category.rb

```diff
@@ -5,10 +5,11 @@ class Category < ActiveRecord::Base
     'none'
   ]
 
-  # TODO(2020-11-18): remove
-  self.ignored_columns = %w{
-    suppress_from_latest
-  }
+  self.ignored_columns = [
+    :suppress_from_latest, # TODO(2020-11-18): remove
```

Comment:
📆💥😛

Reply:
Yeah we should clean it up, but not as part of this PR. Last time someone tried to remove it, things broke: https://github.com/discourse/discourse/commit/6fb89c153a6072ca076f2d46e53a9df1842780a9